Formula/git-recent.rb

```diff
@@ -0,0 +1,16 @@
+class GitRecent < Formula
+  desc "See your latest local git branches, formatted real fancy"
+  homepage "https://github.com/paulirish/git-recent"
+  url "https://raw.githubusercontent.com/paulirish/git-recent/029cc001030d224c961da95502ecee0b54af1989/git-recent"
+  version "1.0.3"
+  sha256 "796dce33edcf6b6031aa0729e443fd4ec46314809bcf3d425d895c90ffe50833"
+
+  def install
+    bin.install "git-recent"
+  end
+
+  test do
+    system "git", "init"
+    system "git", "recent"
```

Comment:
It's a commit that would be needed rather than a branch, but I can do - it's just the output's fairly 'fancy' - it would include the committer name and how many seconds ago the commit was which presumably couldn't be relied upon.

Enough to just `git commit --allow-empty -m 'this is a test'` and check for the text `master` and `this is a test`? Full output would look like (`s/foo/this is a test/`):

![screen shot 2017-01-29 at 16 50 45](https://cloud.githubusercontent.com/assets/3946014/22406050/2d8304ca-e643-11e6-9ffa-1a5a4dd649c3.png)